frontend/src/Components/Items/Items.module.scss

```diff
@@ -0,0 +1,23 @@
+@import "~plaid-threads/scss/variables";
+
+.itemsContainer {
+  width: 100%;
+  border: 1px solid $black200;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  box-shadow: $shadow-small;
+  margin: 3 * $unit 0 3 * $unit 0;
+}
+
+.itemsHeader {
+  width: 100%;
+  border: 1px solid $black200;
+  border-bottom: 0px;
+  height: 13 * $unit;
+  font-weight: 800;
+  padding-top: 5 * $unit;
+  padding-left: 5 * $unit;
+  margin-top: 0;
+  margin-bottom: 0;
```

Comment:
which styles are actually required here? doesnt look like you need margin for any reason